zendesk/organization.go

```diff
@@ -10,16 +10,17 @@ import (
 // Organization is struct for organization payload
 // https://developer.zendesk.com/rest_api/docs/support/organizations
 type Organization struct {
-	ID             int64     `json:"id,omitempty"`
-	URL            string    `json:"url,omitempty"`
-	Name           string    `json:"name"`
-	DomainNames    []string  `json:"domain_names"`
-	GroupID        int64     `json:"group_id"`
-	SharedTickets  bool      `json:"shared_tickets"`
-	SharedComments bool      `json:"shared_comments"`
-	Tags           []string  `json:"tags"`
-	CreatedAt      time.Time `json:"created_at,omitempty"`
-	UpdatedAt      time.Time `json:"updated_at,omitempty"`
+	ID                int64                  `json:"id,omitempty"`
+	URL               string                 `json:"url,omitempty"`
+	Name              string                 `json:"name"`
+	DomainNames       []string               `json:"domain_names"`
+	GroupID           int64                  `json:"group_id"`
+	SharedTickets     bool                   `json:"shared_tickets"`
+	SharedComments    bool                   `json:"shared_comments"`
+	Tags              []string               `json:"tags"`
+	CreatedAt         time.Time              `json:"created_at,omitempty"`
+	UpdatedAt         time.Time              `json:"updated_at,omitempty"`
+	Organizationields map[string]interface{} `json:"organization_fields,omitempty"`
```

Comment:
Fix typo

```suggestion
	OrganizationFields map[string]interface{} `json:"organization_fields,omitempty"`
```

Reply:
And i think that this field should be defied as designated struct

Reply:
I would have liked to use a strut, I cold not see a way to do this. I'd be happy to take on any recommendation. 


For Tickets the custom_fields are in a fixed structure, making is easy to use a strut, however organization_fields do not follow the same patten


```
  "custom_fields": [
    {
      "id":    27642,
      "value": "745"
    },
    {
      "id":    27648,
      "value": "yes"
    }
  ],
```

```
  "organization_fields": {
    "org_dropdown": "option_1",
    "org_decimal": 5.2,
  }
```